evm/scripts/merge.py

```diff
@@ -38,9 +38,9 @@ def main():
         data_file.close()
         evm_results.append(data)
 
-    for i in range(1, len(evm_results[0])):
-        for e in range(0, len(EVMS)):
-            if EVMS[e] == 'evmone':
+    for e in range(0, len(EVMS)):
+        for i in range(0, len(evm_results[e])):
```

Comment:
The other ordering looked nicer from the perspective of being able to compare engines against eachotehr in the CSV.

Reply:
this fixed an error with evm384 running different benchmarks than the other evm engines.